server/app.js

```diff
@@ -1,25 +1,36 @@
 /**
- * Main server code for handling requests from the UI. This should serve some APIs, 
- * the main one being an API that accepts data in order to initiate the workflow for 
+ * Main server code for handling requests from the UI. This should serve some APIs,
+ * the main one being an API that accepts data in order to initiate the workflow for
  * listening to a Twitch channel. The event handler for listening to the twitch events
  * will handle processing the data.
  */
 
 // TODO: figure out how to get dynamic properties set up so that it can run locally
 //       and deployed.
-//       Use an environment variable to define the properties that we want to use. 
+//       Use an environment variable to define the properties that we want to use.
 const properties = {
-    "env": "local"
-}
+  env: "local",
+};
 // const properties = require("./properties.json");
 
 const express = require("express");
 const bodyParser = require("body-parser");
 
-const AWSXRay = require('aws-xray-sdk-core'); // add X-Ray metrics
-const AWS = AWSXRay.captureAWS(require('aws-sdk'));
+const AWSXRay = require("aws-xray-sdk-core"); // add X-Ray metrics
+
+const AWS = AWSXRay.captureAWS(require("aws-sdk"));
+
+AWSXRay.setContextMissingStrategy(() => {});
```

Comment:
Let's get rid of X-Ray since localstack doesn't support it in the community version. We'll survive without it - if needed, we can just add cloudwatch metrics